great_expectations/validation_operators/actions.py

```diff
@@ -10,13 +10,19 @@
 
 logger = logging.getLogger(__name__)
 
+"""
```

Comment:
Module docs need to be at the top of the module (as is, this will show up as a docstring for the logger... :))

![image](https://user-images.githubusercontent.com/332575/85133567-ecc46d80-b208-11ea-8c42-065a25d86794.png)

Reply:
Thank you - addressed!